assignments/boolean-birds/code.js

```diff
@@ -16,3 +16,27 @@
  *   - Globby Birds eat fish, mice, and nuts.
  *
  */
+const isFlobbyBird=(red,spotted) => (red&&spotted)
```

Comment:
You don't need the `()`s around the body expression.

```suggestion
const isFlobbyBird=(red,spotted) => red&&spotted;
```